index.html

```diff
@@ -64,7 +65,254 @@ <h1>I'm Kwamboka Glad to see you!</h1>
+            <div class="about-me">
+                <div>
+                    <h1>About <br> myself</h1>
+                </div>
+                <div>
+                    <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my
+                        work and
+                        experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                </div>
+            </div>
+            <div class="flex-connect">
+                <p>LET’S CONNECT</p>
+            </div>
+            <section class="footer">
+                <ul>
+                    <li>
+                        <a href="#">
+                            <img src="./twitter.png" alt="twitter">
+                        </a>
+                    </li>
+                    <li class="linkedin">
+                        <a href="#">
+                            <img src="./linkedin.png" alt="linkedin">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#">
+                            <img src="./Mail.png" alt="Mail">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#">
+                            <img src="./github.png" alt="github">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#">
+                            <img src="./chill.png" alt="chill">
+                        </a>
+                    </li>
+                </ul>
+            </section>
+            <div class="resume">
+                <button class="Tonic-project resume1" type="button">Get my Resume</button>
+            </div>
+            <div class="language">
+                <ul>
+                    <li>
+                        <p>Languages</p>
+                    </li>
+                    <li class="arrow-down">
+                        <a href="#"><i class="fa fa-angle-down fa-2x" aria-hidden="true"></i></a>
+                    </li>
+                </ul>
+            </div>
+            <div class="lang javascript">
+                <img src="./JS.png" alt="javascript">
+                <span>Javascript</span>
+            </div>
+            <div class="lang HTML">
+                <img src="./HTML.png" alt="HTML"><span>HTML</span>
+            </div>
+            <div class="lang CSS">
+                <img src="./CSS.png" alt="CSS"><span> CSS</span>
+            </div>
+            <div class="Frameworks1">
+                <ul>
+                    <li>
+                        <p>Frameworks</p> 
+                    </li>
+                    <li>
+                        <a href="#"><i class="fa fa-angle-right fa-2x" aria-hidden="true"></i></a>
+                    </li>
+                </ul>
+            </div>
+            <div class="divider">
+            
+            </div>
```

Comment:
- [ ] Kindly remove empty or unnecessary blocks to make the code clean and look professional.

Reply:
I removed the empty spaces. I wanted to create a divider line.